_sections/30-projects.md

```diff
@@ -47,6 +47,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/scikit-image.png)Scikit Image](http://scikit-image.org/) <!-- url:https://github.com/scikit-image/scikit-image sg:3069 -->
 - [![](assets/sphinx-doc.png)Sphinx](https://www.sphinx-doc.org/) <!-- url:https://github.com/sphinx-doc/sphinx sg:2681 -->
 - [![](assets/geopy.png)geopy](https://geopy.readthedocs.io/) <!-- url:https://github.com/geopy/geopy sg:2476 -->
+- [![](https://nnabla.org/logos/logo3.png)Neural Network Libraries](https://nnabla.org/)
```

Comment:
Could you put an HTML comment with the Github URL, like the other lines? There's a script that gets the updated star counts using these and resorts the list.

Reply:
Sure, got it.  Thank you for the feedback.

Reply:
I added GitHub URL.  Please confirm the change thanks.